src/java.base/unix/native/libjava/childproc.c

```diff
@@ -426,6 +426,11 @@ childProcess(void *arg)
         sigprocmask(SIG_SETMASK, &unblock_signals, NULL);
     }
 
+    // Childs should be started with default signal disposition for SIGPIPE
```

Comment:
Editorial: "Childs" -> Children for plural, "Child" for singular.  Here and in tests.